mcstatus/pinger.py

```diff
@@ -218,9 +221,19 @@ def __init__(self, raw):
                 raise ValueError(f"Invalid version object (expected 'protocol' to be int, was {type(raw['protocol'])})")
             self.protocol = raw["protocol"]
 
+    class Description(str):
+        """Just add method `clean` to `description` field."""
+
+        def clean(self) -> str:
+            """Remove colors (&7, &a, §1 etc.) from description.
```

Comment:
```suggestion
            """Get description without color markers (&7, &a, §1 etc.).
```

Also applies for a duplicate class defined in `bedrock_status.py`.

Reply:
Changed with force push.